sematic/BUILD

```diff
@@ -1,4 +1,22 @@
-load("//sematic:wheel_version.bzl", "wheel_version_string")
+load("//sematic:wheel_constants.bzl",
```

Comment:
Moved all the constants for the wheel to one place. It's easier to get to them and change them when they're in a dedicated file rather than having to browse to the correct `BUILD`. Also, there was an earlier refactor where I had two wheel targets and this made it easier :-)